habitat-lab/habitat/config/habitat/simulator/agents/habitat_mas_agents/SpotRobot_default.yaml

```diff
@@ -8,11 +8,11 @@ defaults:
   - /habitat/simulator/sim_sensors@sim_sensors.arm_rgb_sensor: arm_rgb_sensor
   - /habitat/simulator/sim_sensors@sim_sensors.arm_depth_sensor: arm_depth_sensor
   - /habitat/simulator/sim_sensors@sim_sensors.arm_semantic_sensor: arm_semantic_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.head_stereo_left_depth_sensor: head_stereo_left_depth_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.head_stereo_right_depth_sensor: head_stereo_right_depth_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.jaw_rgb_sensor: jaw_rgb_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.jaw_depth_sensor: jaw_depth_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.jaw_semantic_sensor: jaw_semantic_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.head_stereo_left_depth_sensor: head_stereo_left_depth_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.head_stereo_right_depth_sensor: head_stereo_right_depth_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.jaw_rgb_sensor: jaw_rgb_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.jaw_depth_sensor: jaw_depth_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.jaw_semantic_sensor: jaw_semantic_sensor
```

Comment:
Why would you comment out stereo and jaw sensors?

Reply:
I was just to lower the sensors to be demonstrated for convenience. If you need it to be changed back to all sensors, please tell me.

Reply:
Please add them back in the SpotRobot_default.yaml. If people want a spot robot with a different sensor setup, they should use SpotRobot_<sensor_setup>.yaml instead. This is what the config system is designed for.